packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```
@@ -65,32 +64,38 @@
     }
   }
 
-  &:invalid:focus {
+  &:invalid:focus-within {
     outline-color: var(--InvalidFocusOutlineColor);
   }
 
   &::placeholder {
     color: var(--PlaceholderTextColor);
   }
+}
 
-  &[type="search"] {
-    padding-inline-start: calc(
-      var(--IndicatorSize) + var(--IndicatorInsetInlineStart) +
-        var(--SearchPaddingInlineStart)
-    );
-  }
+.decoration {
```

Is there a benefit to using `.decoration` here rather than a combined `.prefix, .suffix` selector?

---

I'm just terrible at css. Can you give me an example of the correct way to do it?

---

No worries! We can re-use the individual selectors here:

```suggestion
.prefix, .suffix {
```

Then you can remove the `.decoration` class from [lines `168` and `170` here](https://github.com/okta/odyssey/pull/1402/files#diff-6a776e2c4a5c8727e55c973d4311de6394251712b8bc55f1e579a3d66277c5b6R168-R170).

Your current approach would work just fine btw! I prefer the joint selector here because it lets us see where the styles are being applied at a glance.

---

Oh 😂 , when I tried that before I was missing the `,` between the selectors. 😆